src/ploomber_core/telemetry/telemetry.py

```diff
@@ -163,7 +163,14 @@ def get_env():
 
 def is_colab():
     """Returns: True for Google Colab env"""
-    return "COLAB_GPU" in os.environ
+    try:
+        import google.colab as colab
+        if colab:
```

Comment:
we don't need this `if colab`. if the package isn't there. this line is never executed.

i'd suggest:

```python
is_colab=True

try:
    import ...
except ModuleNotFoundError:
    is_colab = False


return is_colab
```

Reply:
I added it since the flake8 fails for not using the import
```
./src/ploomber_core/telemetry/telemetry.py:167:9: F401 'google.colab' imported but unused
```

Reply:
you can add `# noqa` to tell flake8 to ignore it